tests/reader-config.rs

```diff
@@ -43,6 +44,73 @@ mod expand_empty_elements {
     }
 }
 
+mod trim_markup_names_in_closing_tags {
+    use super::*;
+    use pretty_assertions::assert_eq;
+
+    mod false_ {
+        use super::*;
+        use pretty_assertions::assert_eq;
+
+        #[test]
+        fn check_end_names_false() {
+            let mut reader = Reader::from_str("<root></root \t\r\n>");
+            reader.trim_markup_names_in_closing_tags(false);
```

Comment:
What is it the reason that we only have this for closing tags, attributes?

Reply:
I think, that this is optimization option. Usually end tags does not contain spaces before `>`, so if we will assume that the name ends immediately before the `>`, we could save some time. Such optimisation make sense only for the closing tags -- for opening tags we in any case should check if it has attributes and find the actual end of tag name.